### PR TITLE
Widget query and naming fixes + widget file hook fixes

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/15022
reference: https://github.com/demisto/etc/issues/16040

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Widgets/widget-IncidentsByRole.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "id": "incidents-by-role",
+  "id": "all-incidents-by-role",
```
Comment (glicht) [inline Widgets/widget-IncidentsByRole.json]: Id change. Please check why.

File: Widgets/widget-ActiveIncidentsByRole.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "id": "mean-time-to-resolution",
+  "id": "incidents-by-role",
```
Comment (glicht) [inline Widgets/widget-ActiveIncidentsByRole.json]: Id change. Please check.

Comment (bakatzir) [reply Widgets/widget-ActiveIncidentsByRole.json]: i deleted the `Mttr` widget, it is not a change of id

**Comment by glicht** [comment]
@bakatzir let's do a test with content update with default dashboard and user dashboard using built-in widgets.

**Comment by bakatzir** [comment]
@glicht 
i uploaded the content manually to my local env:

main dashboard:
wi5

new dashboard:
wi6

code change for this widget:


bottom line:
current widget was not affected by the change, but upon creating new dashboards, only the new are available

**Pull request merged**